src/layouts/EditHomepage.jsx

```diff
@@ -320,10 +331,17 @@ const EditHomepage = ({ match }) => {
     loadPageDetails()
   }, [homepageData])
 
+  const [isFirstLoadComplete, setIsFirstLoadComplete] = useState(false)
```

Comment:
could we shift this into a custom hook? this doesn't seem too sustainable to me because we have a large amount of state in this file - would be more comfortable having this in another hook!

Reply:
Shifted in https://github.com/isomerpages/isomercms-frontend/pull/1439/commits/0ae03dc7afc029741b9e2484d9060ea5b6c8c79b!